onnx/checker.cc

```diff
@@ -274,11 +275,24 @@ void check_node(
   const auto* schema = ctx.get_schema_registry()->GetSchema(
       node.op_type(), domain_version, node.domain());
   if (!schema) {
-    fail_check(
-        "No Schema registered for " + node.op_type() +
-        " with domain_version of " + ONNX_NAMESPACE::to_string(domain_version));
+    GraphProto g;
```

Comment:
where's g from?

Reply:
So the g is a local initialized temp graph, and the ptemp_node is initialized as a copy of the current node (n) we are visiting in the original graph. The g is basically a graph with only the node n. We do this because we dont want to alter the original graph.

Then we apply the checker iteratively on every node in the expanded new graph, with inherited checker_context information from above checked nodes.